### DISCUSSION TITLE
Interface tags aren't being replaced by the chosen language

### DISCUSSION THREAD

**Discussion by jcintra** [requarks/wiki#3500] [Error / Bug Report]
**Describe the bug**
Every tag of the interface isn't being replaced, some examples on the administration page:
* dashboard.title
* dashboard.subtitle
* DASHBOARD.PAGES
* DASHBOARD.USERS
* DASHBOARD.GROUPS
...

**To Reproduce**
I used a docker container and installed it as a stack of portainer.io using the following script:
---
version: "2.1"
services:
  wikijs:
    image: linuxserver/wikijs
    container_name: wikijs
    environment:
      - PUID=0
      - PGID=0
      - TZ=America/Denver
    volumes:
      - /srv/WikiJS/config:/config
      - /srv/WikiJS/data:/data
    ports:
      - 3000:3000
    restart: unless-stopped

**Expected behavior**
The tags should be filled with English names.

**Additional context**
Everytime I change pages I have an error:
"An unexpected error occurred."

**Comment by Smankusors** [comment]
where's the host info?

---

on the other hand, is your wiki.js on offline mode?

**Comment by jcintra** [comment]
Hello Smankusors,

I installed the wiki.js on my machine. I don't know what is offline mode. I followed a video where the guy used also the Portainer and it worked without a problem. Do you know where to find the logs? Maybe I can get a better look at the real problem.
It seems that wiki.js isn't loading a locale file and I'm just getting the tags.

Can you help me with understanding what am I missing?
Thanks

**Comment by Smankusors** [comment]
btw are you selecting english or different language? It's possible that the translation of your choosen language is not complete yet, and by default it's not fallback to the english.

**Comment by jcintra** [comment]
It was english.
Is there a way to put the missing file manually in the right place?

**Comment by calebcbaker** [comment]
I'm getting the same thing but I'm running directly on Windows Server 2019 using SQlite as the database, NodeJS v 12.18.3. It should also be noted that I'm running behind a corporate firewall so outside access to auto download locales is a no go for me.

**Comment by Smankusors** [comment]
I think you can follow [this guide here](https://docs.requarks.io/install/sideload).

**Comment by nacho-milanese** [comment]
I had this issue. I've solved it by:
- downloading a new locale.
- switch to that locale (by this time it has been solved)
- switched back to english (and the error wasn't there anymore)
